argoverse/evaluation/stereo/disparity_interpolation/disparity_interpolation/source/nn_interpolation.cpp

```diff
@@ -0,0 +1,80 @@
+#include "nn_interpolation.h"
+#include <stdint.h>
+
```

Comment:
maybe a 3-4 line docstring at the top of this file could describe the high-level algorithm/pseudocode?

Reply:
It's on pypi now.

Reply:
Wow that was fast! Really nice work.

I couldnt access https://github.com/jhonykaesemodel/disparity_interpolation though -- is that not pushed quite yet?

Reply:
Not published on my github yet. Will do it soon.

Reply:
Published now.

Reply:
Cool. Maybe you could add a high-level algorithm overview to that other repo, and a few small unit tests on toy data too?

Reply:
Definitely, already in my to-do list.